components/builder-web/package-lock.json

```diff
@@ -9,98 +9,98 @@
       "resolved": "https://registry.npmjs.org/@angular/animations/-/animations-6.1.10.tgz",
       "integrity": "sha512-dd/lq7kw3uwfHPICan8psu2nthuUpp7PvMLuNIm0XxObZ4oNs0ls6uxKEDPnEkRKoGdiJpvmsyzZZN9ACMPEAA==",
       "requires": {
-        "tslib": "^1.9.0"
```

Comment:
do we want to be making all these updates?

Reply:
Will be rolled back with the next push.